readme.md

```diff
@@ -1468,7 +1468,7 @@ One of the most well known scifi films, this is a great expose on what people th
 
 Categories: `remake` `steampunk`
 
-> Hoping to alter the events of the past, a 19th century inventor instead travels 800,000 years into the future, where he finds humankind divided into two warring races.
+> Hoping to alter the events of the past, a 19th century inventor instead travels 800,000 years into the future, where he finds humankind divided into two warring races. - [@damaru2](https://github.com/damaru2)
```

Comment:
This is the description, not a comment for this movie - not sure the link is right here.